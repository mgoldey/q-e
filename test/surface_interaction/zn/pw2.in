&CONTROL
  calculation  = "scf",
  prefix       = "left",
  outdir       = "./out",
  pseudo_dir   = "../../pseudo",
  verbosity = "high" 
  wf_collect = .true.
/
&SYSTEM
  nosym = .true.
  nbnd = 14
  ibrav = 1,
  a = 12,
  nat = 2,
  ntyp = 1,
  ecutwfc = 50,
  tot_charge = +1
  nspin = 2,
  tot_magnetization = 1
/
&ELECTRONS
 ! startingpot = 'file'
 ! startingwfc = 'file'
  conv_thr    = 1.D-7,
  mixing_beta = 0.30,
/
EPCDFT
1 1.D+2 1.D-2 100 .true.
delta_charge 1 1 2 2 0.0 0.0

K_POINTS {Gamma}

ATOMIC_SPECIES
Zn 0.0 Zn.pbe-mt_fhi.UPF
ATOMIC_POSITIONS (angstrom)
Zn 5 5 4
Zn 5 5 2
