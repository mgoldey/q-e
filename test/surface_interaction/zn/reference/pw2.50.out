
     Program PWSCF v.5.1.1 starts on 10May2016 at  8:52:12 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI), running on     4 processors
     R & G space division:  proc/nbgrp/npool/nimage =       4
     Waiting for input...
     Reading input from standard input
 Reading EPCDFT CARD. Printing based upon belief that input should be reproducable from output.
     Reading   1 constraint(s); tolerance:  100.000000 max change in field 0.01
     epcdft_update_intrvl: 100
     epcdft with surface: T
          constraint           type   A   A   D   D      T      L
                   1  delta_charge    1   1   2   2 0.0000 0.0000

     Current dimensions of program PWSCF are:
     Max number of different atomic species (ntypx) = 10
     Max number of k-points (npk) =  40000
     Max angular momentum in pseudopotentials (lmaxx) =  3
               file Zn.pbe-mt_fhi.UPF: wavefunction(s)  4p 4f renormalized

     gamma-point specific algorithms are used
     Message from routine setup:
     the system is metallic, specify occupations

     Subspace diagonalization in iterative solution of the eigenvalue problem:
     a serial algorithm will be used


     Parallelization info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Min        2047    2047    512               139165   139165   17446
     Max        2052    2052    514               139168   139168   17450
     Sum        8197    8197   2053               556667   556667   69791
     Tot        4099    4099   1027

     Generating pointlists ...
     new r_m :   0.0687 (alat units)  1.5590 (a.u.) for type    1


     bravais-lattice index     =            1
     lattice parameter (alat)  =      22.6767  a.u.
     unit-cell volume          =   11661.1222 (a.u.)^3
     number of atoms/cell      =            2
     number of atomic types    =            1
     number of electrons       =        23.00 (up:  12.00, down:  11.00)
     number of Kohn-Sham states=           14
     kinetic-energy cutoff     =      50.0000  Ry
     charge density cutoff     =     200.0000  Ry
     convergence threshold     =      1.0E-07
     mixing beta               =       0.3000
     number of iterations used =            8  plain     mixing
     Exchange-correlation      = PBE ( 1  4  3  4 0 0)

     celldm(1)=  22.676714  celldm(2)=   0.000000  celldm(3)=   0.000000
     celldm(4)=   0.000000  celldm(5)=   0.000000  celldm(6)=   0.000000

     crystal axes: (cart. coord. in units of alat)
               a(1) = (   1.000000   0.000000   0.000000 )  
               a(2) = (   0.000000   1.000000   0.000000 )  
               a(3) = (   0.000000   0.000000   1.000000 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = (  1.000000  0.000000  0.000000 )  
               b(2) = (  0.000000  1.000000  0.000000 )  
               b(3) = (  0.000000  0.000000  1.000000 )  


     PseudoPot. # 1 for Zn read from file:
     ../../pseudo/Zn.pbe-mt_fhi.UPF
     MD5 check sum: 5596c9f2b89b9f7367257e7ea958b26e
     Pseudo is Norm-conserving, Zval = 12.0
     Generated using FHI98PP, converted with fhi2upf.x v.5.0.2
     Using radial grid of  527 points,  3 beta functions with: 
                l(1) =   1
                l(2) =   2
                l(3) =   3

     atomic species   valence    mass     pseudopotential
        Zn            12.00    65.39000     Zn( 1.00)

     Starting magnetic structure 
     atomic species   magnetization
        Zn           0.000

     No symmetry found


                                    s                        frac. trans.

      isym =  1     identity                                     

 cryst.   s( 1) = (     1          0          0      )
                  (     0          1          0      )
                  (     0          0          1      )

 cart.    s( 1) = (  1.0000000  0.0000000  0.0000000 )
                  (  0.0000000  1.0000000  0.0000000 )
                  (  0.0000000  0.0000000  1.0000000 )


     point group C_1 (1)    
     there are  1 classes
     the character table:

       E    
A      1.00

     the symmetry operations in each class:
     E        1

   Cartesian axes

     site n.     atom                  positions (alat units)
         1           Zn  tau(   1) = (   0.4166667   0.4166667   0.3750000  )
         2           Zn  tau(   2) = (   0.4166667   0.4166667   0.2083333  )

   Crystallographic axes

     site n.     atom                  positions (cryst. coord.)
         1           Zn  tau(   1) = (  0.4166667  0.4166667  0.3750000  )
         2           Zn  tau(   2) = (  0.4166667  0.4166667  0.2083333  )

     number of k points=     2
                       cart. coord. in units 2pi/alat
        k(    1) = (   0.0000000   0.0000000   0.0000000), wk =   1.0000000
        k(    2) = (   0.0000000   0.0000000   0.0000000), wk =   1.0000000

                       cryst. coord.
        k(    1) = (   0.0000000   0.0000000   0.0000000), wk =   1.0000000
        k(    2) = (   0.0000000   0.0000000   0.0000000), wk =   1.0000000

     Dense  grid:   278334 G-vectors     FFT dimensions: ( 108, 108, 108)

     Largest allocated arrays     est. size (Mb)     dimensions
        Kohn-Sham Wavefunctions         1.86 Mb     (    8725,   14)
        NL pseudopotentials             3.99 Mb     (    8725,   30)
        Each V/rho on FFT grid          9.61 Mb     (  314928,   2)
        Each G-vector array             0.53 Mb     (   69584)
        G-vector shells                 0.02 Mb     (    2154)
     Largest temporary arrays     est. size (Mb)     dimensions
        Auxiliary wavefunctions         3.73 Mb     (    8725,   56)
        Each subspace H/S matrix        0.02 Mb     (      56,   56)
        Each <psi_i|beta_j> matrix      0.00 Mb     (      30,   14)
        Arrays for rho mixing          38.44 Mb     (  314928,    8)

     Initial potential from superposition of free atoms
     Check: negative starting charge=(component1):   -0.041678
     Check: negative starting charge=(component2):   -0.041678

     starting charge   23.99975, renormalised to   23.00000

     negative rho (up, down):  3.994E-02 3.994E-02
     
     Adding image charge field due to neutral metal slab at origin in XY plane.
     Including monopole and dipole terms.
     Quadrupole not implemented yet.
     System should not overlap with cell edges.
     

     reference position (x0):         9.44863066    9.44863066    6.61404147 bohr
     Total charge   1.0000
     Total dipole moment   0.0004   0.0004  -0.0000 au (Ha),   0.0010   0.0010  -0.0000 Debye

     Debug on write surface field to cube file.

     First surface image interaction energy:-0.779E-01

     
     Adding image charge field due to neutral metal slab at origin in XY plane.
     Including monopole and dipole terms.
     Quadrupole not implemented yet.
     System should not overlap with cell edges.
     

     reference position (x0):         9.44863066    9.44863066    6.61404147 bohr
     Total charge   1.0000
     Total dipole moment   0.0004   0.0004  -0.0000 au (Ha),   0.0010   0.0010  -0.0000 Debye

     Debug on write surface field to cube file.
     Starting wfc are   32 randomized atomic wfcs

     total cpu time spent up to now is        6.7 secs

     per-process dynamical memory:    78.3 Mb

     Self-consistent Calculation

     iteration #  1     ecut=    50.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  1.00E-02,  avg # of iterations =  4.0

     negative rho (up, down):  2.726E-02 2.751E-02

     Magnetic moment per site:
     atom:    1    charge:    9.3419    magn:    0.0389    constr:    0.0000
     atom:    2    charge:    9.3341    magn:    0.0374    constr:    0.0000

     total cpu time spent up to now is       10.2 secs

     total energy              =    -217.97574223 Ry
     Harris-Foulkes estimate   =    -218.17245085 Ry
     estimated scf accuracy    <       0.53908620 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.00 Bohr mag/cell

     iteration #  2     ecut=    50.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  2.34E-03,  avg # of iterations =  1.5

     negative rho (up, down):  1.241E-02 1.277E-02

     Magnetic moment per site:
     atom:    1    charge:    9.4642    magn:    0.0774    constr:    0.0000
     atom:    2    charge:    9.4572    magn:    0.0762    constr:    0.0000

     total cpu time spent up to now is       13.1 secs

     total energy              =    -217.96629201 Ry
     Harris-Foulkes estimate   =    -218.01291178 Ry
     estimated scf accuracy    <       0.11801822 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.01 Bohr mag/cell

     iteration #  3     ecut=    50.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  5.13E-04,  avg # of iterations =  2.0

     negative rho (up, down):  6.639E-03 6.983E-03

     Magnetic moment per site:
     atom:    1    charge:    9.4583    magn:    0.0855    constr:    0.0000
     atom:    2    charge:    9.4555    magn:    0.0851    constr:    0.0000

     total cpu time spent up to now is       16.8 secs

     total energy              =    -217.99303092 Ry
     Harris-Foulkes estimate   =    -217.99162579 Ry
     estimated scf accuracy    <       0.00963333 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.04 Bohr mag/cell

     iteration #  4     ecut=    50.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  4.19E-05,  avg # of iterations =  2.0

     negative rho (up, down):  2.570E-05 2.662E-05

     Magnetic moment per site:
     atom:    1    charge:    9.4772    magn:    0.0978    constr:    0.0000
     atom:    2    charge:    9.4753    magn:    0.0984    constr:    0.0000

     total cpu time spent up to now is       19.8 secs

     total energy              =    -217.99457071 Ry
     Harris-Foulkes estimate   =    -217.99388023 Ry
     estimated scf accuracy    <       0.00275919 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.05 Bohr mag/cell

     iteration #  5     ecut=    50.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  1.20E-05,  avg # of iterations =  2.5

     negative rho (up, down):  2.191E-05 2.264E-05

     Magnetic moment per site:
     atom:    1    charge:    9.4710    magn:    0.0956    constr:    0.0000
     atom:    2    charge:    9.4690    magn:    0.0962    constr:    0.0000

     total cpu time spent up to now is       22.9 secs

     total energy              =    -217.99510342 Ry
     Harris-Foulkes estimate   =    -217.99513592 Ry
     estimated scf accuracy    <       0.00035054 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.08 Bohr mag/cell

     iteration #  6     ecut=    50.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  1.52E-06,  avg # of iterations =  6.0

     Magnetic moment per site:
     atom:    1    charge:    9.4669    magn:    0.0931    constr:    0.0000
     atom:    2    charge:    9.4647    magn:    0.0937    constr:    0.0000

     total cpu time spent up to now is       26.2 secs

     total energy              =    -217.99512897 Ry
     Harris-Foulkes estimate   =    -217.99506400 Ry
     estimated scf accuracy    <       0.00003665 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.08 Bohr mag/cell

     iteration #  7     ecut=    50.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  1.59E-07,  avg # of iterations =  3.0

     Magnetic moment per site:
     atom:    1    charge:    9.4674    magn:    0.0926    constr:    0.0000
     atom:    2    charge:    9.4652    magn:    0.0933    constr:    0.0000

     total cpu time spent up to now is       29.7 secs

     total energy              =    -217.99513638 Ry
     Harris-Foulkes estimate   =    -217.99513300 Ry
     estimated scf accuracy    <       0.00001153 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.09 Bohr mag/cell

     iteration #  8     ecut=    50.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  5.01E-08,  avg # of iterations =  2.0

     Magnetic moment per site:
     atom:    1    charge:    9.4675    magn:    0.0924    constr:    0.0000
     atom:    2    charge:    9.4653    magn:    0.0931    constr:    0.0000

     total cpu time spent up to now is       32.7 secs

     total energy              =    -217.99513091 Ry
     Harris-Foulkes estimate   =    -217.99513716 Ry
     estimated scf accuracy    <       0.00000130 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.09 Bohr mag/cell

     iteration #  9     ecut=    50.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  5.63E-09,  avg # of iterations =  2.5

     Magnetic moment per site:
     atom:    1    charge:    9.4677    magn:    0.0924    constr:    0.0000
     atom:    2    charge:    9.4655    magn:    0.0930    constr:    0.0000

     total cpu time spent up to now is       36.0 secs

     total energy              =    -217.99513584 Ry
     Harris-Foulkes estimate   =    -217.99513679 Ry
     estimated scf accuracy    <       0.00000030 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.09 Bohr mag/cell

     iteration # 10     ecut=    50.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  1.30E-09,  avg # of iterations =  2.5
     
     Adding image charge field due to neutral metal slab at origin in XY plane.
     Including monopole and dipole terms.
     Quadrupole not implemented yet.
     System should not overlap with cell edges.
     

     reference position (x0):         9.44863066    9.44863066    6.61404147 bohr
     Total charge   1.0000
     Total dipole moment  -0.0004  -0.0004  -0.2240 au (Ha),  -0.0010  -0.0010  -0.5695 Debye

     Debug on write surface field to cube file.
       I         D           A         Val      Target            Str
       1 0.109E+01   0.103E+01   0.544E-01   0.000E+00   0.000000E+00
     Image interaction energy: -0.780E-01 Change: -0.154E-03 Tolerance:  0.100E-04

     Magnetic moment per site:
     atom:    1    charge:    9.4677    magn:    0.0924    constr:    0.0000
     atom:    2    charge:    9.4655    magn:    0.0930    constr:    0.0000

     total cpu time spent up to now is       40.9 secs

     total energy              =    -217.99513650 Ry
     Harris-Foulkes estimate   =    -217.99513757 Ry
     estimated scf accuracy    <       0.00000008 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.09 Bohr mag/cell

     iteration # 11     ecut=    50.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  3.31E-10,  avg # of iterations =  1.0
     
     Adding image charge field due to neutral metal slab at origin in XY plane.
     Including monopole and dipole terms.
     Quadrupole not implemented yet.
     System should not overlap with cell edges.
     

     reference position (x0):         9.44863066    9.44863066    6.61404147 bohr
     Total charge   1.0000
     Total dipole moment  -0.0004  -0.0004  -0.2240 au (Ha),  -0.0010  -0.0010  -0.5695 Debye

     Debug on write surface field to cube file.
       I         D           A         Val      Target            Str
       1 0.109E+01   0.103E+01   0.544E-01   0.000E+00   0.000000E+00
     Image interaction energy: -0.780E-01 Change:  0.495E-09 Tolerance:  0.100E-04
     
     Adding image charge field due to neutral metal slab at origin in XY plane.
     Including monopole and dipole terms.
     Quadrupole not implemented yet.
     System should not overlap with cell edges.
     

     reference position (x0):         9.44863066    9.44863066    6.61404147 bohr
     Total charge   1.0000
     Total dipole moment  -0.0004  -0.0004  -0.2240 au (Ha),  -0.0010  -0.0010  -0.5695 Debye

     Debug on write surface field to cube file.

     Magnetic moment per site:
     atom:    1    charge:    9.4676    magn:    0.0924    constr:    0.0000
     atom:    2    charge:    9.4654    magn:    0.0931    constr:    0.0000

     total cpu time spent up to now is       46.3 secs

     End of self-consistent calculation

 ------ SPIN UP ------------


          k = 0.0000 0.0000 0.0000 ( 34896 PWs)   bands (ev):

   -14.4409   -13.7249   -13.7249   -12.9555   -12.9511   -12.7232   -12.7065   -12.4900
   -12.4900   -12.4476   -10.4962    -6.6297    -3.8722    -3.0125
     occupation numbers 
     1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000
     1.0000   1.0000   1.0000   1.0000   0.0000   0.0000

 ------ SPIN DOWN ----------


          k = 0.0000 0.0000 0.0000 ( 34896 PWs)   bands (ev):

   -14.3087   -13.6155   -13.6155   -12.8775   -12.8729   -12.6439   -12.6274   -12.3355
   -12.3355   -12.2605   -10.2514    -5.7063    -3.5190    -3.5190
     occupation numbers 
     1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000
     1.0000   1.0000   1.0000   0.0000   0.0000   0.0000

     highest occupied level (ev):    -6.6297
     
     Adding image charge field due to neutral metal slab at origin in XY plane.
     Including monopole and dipole terms.
     Quadrupole not implemented yet.
     System should not overlap with cell edges.
     

     reference position (x0):         9.44863066    9.44863066    6.61404147 bohr
     Total charge   1.0000
     Total dipole moment  -0.0004  -0.0004  -0.2240 au (Ha),  -0.0010  -0.0010  -0.5695 Debye

     Debug on write surface field to cube file.
       I         D           A         Val      Target            Str
       1 0.109E+01   0.103E+01   0.544E-01   0.000E+00   0.000000E+00
     Image interaction energy: -0.780E-01 Change:  0.495E-09 Tolerance:  0.100E-04

!    total energy              =    -217.99512568 Ry
     Harris-Foulkes estimate   =    -217.99513757 Ry
     estimated scf accuracy    <       0.00000008 Ry

     The total energy is the sum of the following terms:

     one-electron contribution =    -400.00325724 Ry
     hartree contribution      =     209.59479183 Ry
     xc contribution           =     -32.48728740 Ry
     ewald contribution        =       4.90223735 Ry
     CDFT correction           =      -0.00000000 Ry
     scf correction            =      -0.00161023 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.09 Bohr mag/cell

     convergence has been achieved in  11 iterations

     Writing output data file left.save

     init_run     :      5.82s CPU      6.66s WALL (       1 calls)
     electrons    :     38.04s CPU     41.38s WALL (       1 calls)

     Called by init_run:
     wfcinit      :      1.35s CPU      1.76s WALL (       1 calls)
     potinit      :      3.82s CPU      4.22s WALL (       1 calls)

     Called by electrons:
     c_bands      :     15.32s CPU     16.59s WALL (      11 calls)
     sum_band     :      4.14s CPU      4.52s WALL (      11 calls)
     v_of_rho     :     10.86s CPU     11.82s WALL (      11 calls)
     v_h          :      0.34s CPU      0.38s WALL (      11 calls)
     v_xc         :     10.52s CPU     11.44s WALL (      11 calls)
     mix_rho      :      1.45s CPU      1.66s WALL (      11 calls)

     Called by c_bands:
     init_us_2    :      0.35s CPU      0.39s WALL (      46 calls)
     regterg      :     14.62s CPU     15.81s WALL (      22 calls)

     Called by sum_band:

     Called by *egterg:
     h_psi        :     14.49s CPU     15.96s WALL (      82 calls)
     g_psi        :      0.06s CPU      0.06s WALL (      58 calls)
     rdiaghg      :      0.03s CPU      0.04s WALL (      80 calls)
     regterg:over :      0.49s CPU      0.52s WALL (      58 calls)
     regterg:upda :      0.29s CPU      0.31s WALL (      58 calls)
     regterg:last :      0.12s CPU      0.12s WALL (      22 calls)

     Called by h_psi:
     h_psi:vloc   :     13.69s CPU     15.09s WALL (      82 calls)
     h_psi:vnl    :      0.73s CPU      0.79s WALL (      82 calls)
     add_vuspsi   :      0.32s CPU      0.35s WALL (      82 calls)

     General routines
     calbec       :      0.41s CPU      0.44s WALL (      82 calls)
     fft          :      7.26s CPU      7.95s WALL (     292 calls)
     fftw         :     13.75s CPU     15.19s WALL (    1030 calls)

     Parallel routines
     fft_scatter  :     11.08s CPU     12.11s WALL (    1322 calls)

     PWSCF        :    44.10s CPU        48.32s WALL


   This run was terminated on:   8:53: 0  10May2016            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
